src/main/java/spring/intro/controller/UserController.java

```diff
@@ -3,33 +3,33 @@
 import java.util.List;
 import java.util.stream.Collectors;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 import spring.intro.dto.UserResponseDto;
 import spring.intro.model.User;
 import spring.intro.service.UserService;
 
 @RestController
-@RequestMapping(value = "/user", method = RequestMethod.GET)
+@RequestMapping("/user")
 public class UserController {
     @Autowired
     private UserService userService;
 
-    @RequestMapping("/")
+    @GetMapping("/")
```

Comment:
тут можна видалити ("/"). Воно буде працювати без цього
```suggestion
    @GetMapping
```

Reply:
done